src/ui/public/index_patterns/_index_pattern.js

```diff
@@ -110,7 +113,11 @@ export default function IndexPatternFactory(Private, Notifier, config, kbnIndex,
 
     if (!indexPattern.fields || !containsFieldCapabilities(indexPattern.fields)) {
       promise = indexPattern.refreshFields();
+    } else if (indexPattern.metaFieldsVersion <= 0) {
```

Comment:
Ok, changed to check for false. Unfortunately had to change the type to 'boolean' as having issue with 'number'. Getting this error when index is first created: [mapper_parsing_exception] No handler for type [number] declared on field [metaFieldsVersion]

Any idea what is the issue is?